cmd/webhook/app/options/options.go

```diff
@@ -75,3 +79,15 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 
 	o.ProfileOpts.AddFlags(flags)
 }
+
+// AddCustomGlobalFlags explicitly registers flags that internal packages register
+// against the global flagsets from "flag".
+func (o *Options) AddCustomGlobalFlags(flags *pflag.FlagSet) {
+	// Lookup flags in global flag set and re-register the values with our flagset.
+
+	// Adds flags from k8s.io/apiserver/pkg/admission.
+	globalflag.Register(flags, "default-propagation-policy-not-ready-toleration-seconds")
+	globalflag.Register(flags, "default-propagation-policy-unreachable-toleration-seconds")
+	globalflag.Register(flags, "default-cluster-propagation-policy-not-ready-toleration-seconds")
+	globalflag.Register(flags, "default-cluster-propagation-policy-unreachable-toleration-seconds")
```

Comment:
IMOP, can we not distinguish between `PropagationPolicy` and `ClusterPropagationPolicy`?

Reply:
Good idea. I also think distinguishing may be redundant.